### PR TITLE
Merge release/18.8 to develop (Release Notes + 18.8-rc-2)

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] Update Release Notes with Editorial'd copy
 - [x] Also includes the recent fix #15647 to update kotlin and make the code compile again

[EDIT]
 - [x] Now also includes #15624 (which got merged after this PR was opened).
 - [x] And version bump + latest translations from the new beta I triggered to include that fix

### CODE REVIEW

**Comment by renanferrari** [comment]
Hey @AliSoftware, would it be possible to add another line into the release notes regarding the About screen? We can reuse the same editorial copy from https://github.com/wordpress-mobile/WordPress-iOS/pull/17619: `We also added a new About screen so you can rate and share the app (hint, hint), visit our Twitter profile and blog, view our other apps, and more.`

This was supposed to be added with #15624, but I just realize it wasn't 😔

File: WordPress/metadata/release_notes.txt [hunk 1]
```diff
@@ -1,12 +1,6 @@
...
-* [**] Block editor: Gallery block: When a gallery block is added, the media options are auto opened for v2 of the Gallery block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4277]
-* [*] Block editor: Media & Text block: Fix an issue where the text font size would be bigger than expected in some cases. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4252]
-
-
+- Get link suggestions instead of auto-populated clipboard links for Button and Image blocks.
+- Click "Copy Link" to share published post links.
+- Inserter menu tabs match your device's language.
+- Device media options open when creating a Gallery block.
+- Font size bug and RTL visual issues are resolved.
+- Edit Embed block link in Block Settings.
```
Comment (renanferrari) [inline WordPress/metadata/release_notes.txt:6]:
```suggestion
- Edit Embed block link in Block Settings.
- We also added a new About screen so you can rate and share the app (hint, hint), visit our Twitter profile, view our other apps, and more.
```

File: WordPress/metadata/PlayStoreStrings.po [hunk 2]
```diff
@@ -11,22 +11,24 @@ msgstr ""
...
 #. translators: Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
+msgctxt "release_note_188"
+msgid ""
+"18.8:\n"
+"- Get link suggestions instead of auto-populated clipboard links for Button and Image blocks.\n"
+"- Click "Copy Link" to share published post links.\n"
+"- Inserter menu tabs match your device's language.\n"
+"- Device media options open when creating a Gallery block.\n"
+"- Font size bug and RTL visual issues are resolved.\n"
+"- Edit Embed block link in Block Settings.\n"
```
Comment (renanferrari) [inline WordPress/metadata/PlayStoreStrings.po:22]:
```suggestion
"- Edit Embed block link in Block Settings.\n"
"- We also added a new About screen so you can rate and share the app (hint, hint), visit our Twitter profile, view our other apps, and more.\n"
```

**Comment by AliSoftware** [comment]
Sadly it won't be possible:
- We're already at the limit in terms of number of characters for the Android release notes so adding anything would made them too long for translation.
- And it's also now too late to send them back to our freelance for editorial revision in the hope they could reword everything shorter just to fit this as this has to land before EOD anyway in order to be sent in time to our translation vendor.

**Review by renanferrari — COMMENTED** [review]
@AliSoftware Just saw your comment above, feel free to ignore this. It's okay, we can follow the original plan to add the release note in the next version. Thanks for letting me know! 

~The About screen is only being added for WordPress (not Jetpack), so I believe these are the changes we would need to make:~

File: WordPress/metadata/release_notes.txt (hunk 1 above)
Comment (AliSoftware) [reply WordPress/metadata/release_notes.txt:6]: This would make the notes exceed their max length allowed for translation and PlayStore 🙁

**Review by jkmassel — APPROVED** [review]

**Pull request merged**